### DISCUSSION TITLE
additional arguments  in weight aggregation functions (Task 1)

### DISCUSSION THREAD

**Discussion by dskhanirfan** [FETS-AI/Challenge#36] [General]
Are we allowed to include additional arguments in custom weight aggregation functions? Other than (local_tensors,
                                 db_iterator,
                                 tensor_name,
                                 fl_round,
                                 collaborators_chosen_each_round,
                                 collaborator_times_per_round)

**Comment by sarthakpati** [comment]
I believe that is not possible at this time. Tagging @brandon-edwards @msheller @psfoley @alexey-gruzdev for additional clarity.
